src/components/Button.vue

```diff
@@ -0,0 +1,35 @@
+
+<template>
```

Comment:
Can we create a more generic button component that supports loading state, disabled state as well?